gcp_variant_transforms/transforms/vcf_field_conflict_resolver_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2018 Google Inc.  All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Test cases for get_merged_headers module."""
+
+import unittest
+import vcf
+
+from gcp_variant_transforms.transforms import vcf_field_conflict_resolver
+
+
+class ConflictResolverTest(unittest.TestCase):
+  """Test case for :class:`FieldConflictResolver`"""
+
+  def setUp(self):
+    self._resolver = vcf_field_conflict_resolver.FieldConflictResolver()
+    self._resolver_allele = vcf_field_conflict_resolver.FieldConflictResolver(
+        split_alternate_allele_info_fields=True)
+
+  def _field_count(self, symbol):
+    # type: (str) -> int
+    # symbol = {'A', 'G', 'R'}.
+    return vcf.parser.field_counts[symbol]
+
+  def test_resolving_field_definition_conflict_in_type(self):
+    self.assertEqual(
+        self._resolver.resolve('type', 'Integer', 'Float'), 'Float')
+    with self.assertRaises(ValueError):
+      self._resolver.resolve('type', 'Integer', 'String')
+      self.fail('Should raise exception for unresolvable types')
+
+  def test_resolving_field_definition_conflict_in_number(self):
+    self.assertEqual(
+        self._resolver.resolve('num', 2, 3), None)
```

Comment:
Maybe 'type' and 'num' should be constants too (and used in test)?

Reply:
Done!